mods/species/tajaran/clothing.dm

```diff
@@ -0,0 +1,145 @@
+#define TAJ_OBJ 'mods/species/tajaran/icons/clothing/obj.dmi'
+#define TAJ_MOB 'mods/species/tajaran/icons/clothing/onmob.dmi'
+
+//Clothing!//
+
+//Veils
+
+/obj/item/clothing/glasses/tajblind
```

Comment:
I'm not sold on porting all the visors, they're going to need to be converted to the Nebula single icon system and frankly I always hated them visually and with regards to how they were shoehorned into the lore to make Taj more distinctive. Possibly something for maintainer discussion.